clrfeatures.cmake

```diff
@@ -2,6 +2,12 @@ if(CLR_CMAKE_TARGET_TIZEN_LINUX)
   set(FEATURE_EVENT_TRACE 0)
 endif()
 
+if(FEATURE_STANDALONE_GC_ONLY)
+  set(FEATURE_EVENT_TRACE 0)
+  set(FEATURE_APPDOMAIN_RESOURCE_MONITORING 0)
+  set(FEATURE_COMINTEROP 0)
```

Comment:
This is an artifact of earlier iterations of this PR, disabling `FEATURE_COMINTEROP` is not necessary to get the GC to build (and in fact, the VM itself doesn't build without it). I will be removing this in the next commit. Even now, this setting doesn't do anything (the `FEATURE_COMINTEROP` define is defined regardless of these setting of the `FEATURE_COMINTEROP` cmake define.